pt-BR/api_app.md

```diff
@@ -32,95 +32,98 @@ muitos desenvolvedores estão tratando suas aplicações web apenas como uma API
 Esse guia aborda a construção de um aplicativo Rails que fornece dados em JSON para um cliente, incluindo frameworks *client-side*.
 
 
-Why Use Rails for JSON APIs?
+Por Que Usar Rails para APIs JSON?
 ----------------------------
 
-The first question a lot of people have when thinking about building a JSON API
-using Rails is: "isn't using Rails to spit out some JSON overkill? Shouldn't I
-just use something like Sinatra?".
-
-For very simple APIs, this may be true. However, even in very HTML-heavy
-applications, most of an application's logic lives outside of the view
-layer.
-
-The reason most people use Rails is that it provides a set of defaults that
-allows developers to get up and running quickly, without having to make a lot of trivial
-decisions.
-
-Let's take a look at some of the things that Rails provides out of the box that are
-still applicable to API applications.
-
-Handled at the middleware layer:
-
-- Reloading: Rails applications support transparent reloading. This works even if
-  your application gets big and restarting the server for every request becomes
-  non-viable.
-- Development Mode: Rails applications come with smart defaults for development,
-  making development pleasant without compromising production-time performance.
-- Test Mode: Ditto development mode.
-- Logging: Rails applications log every request, with a level of verbosity
-  appropriate for the current mode. Rails logs in development include information
-  about the request environment, database queries, and basic performance
-  information.
-- Security: Rails detects and thwarts [IP spoofing
-  attacks](https://en.wikipedia.org/wiki/IP_address_spoofing) and handles
-  cryptographic signatures in a [timing
-  attack](https://en.wikipedia.org/wiki/Timing_attack) aware way. Don't know what
-  an IP spoofing attack or a timing attack is? Exactly.
-- Parameter Parsing: Want to specify your parameters as JSON instead of as a
-  URL-encoded String? No problem. Rails will decode the JSON for you and make
-  it available in `params`. Want to use nested URL-encoded parameters? That
-  works too.
-- Conditional GETs: Rails handles conditional `GET` (`ETag` and `Last-Modified`)
-  processing request headers and returning the correct response headers and status
-  code. All you need to do is use the
+A primeira questão que muitas pessoas têm quando estão pensando em construir uma
+API JSON utilizando Rails é: "Utilizar Rails para retornar alguns JSON não é
+exagero? Não deveríamos usar algo como Sinatra?"
+
+Para APIs muito simples, isso pode ser verdade. Porém, até mesmo em aplicações
+com muito HTML, boa parte da lógica de uma aplicação está fora da camada de
+visualização.
+
+A razão da maioria das pessoas usar o Rails é que ele fornece um conjunto de
+padrões que permitem desenvolvedores criarem e rodarem rápido, sem ter de fazer
+muitas decisões triviais.
+
+Vamos dar uma olhada em algumas das coisas que o Rails fornece "fora da caixa"
```

Comment:
Como "out of the box" é uma expressão muito particular da língua inglesa, que acha de adaptar pra algo tipo "direto da caixa" ou "diretamente da caixa"? Acho que uma pessoa que não está acostumada com inglês pode confundir com outra expressão tipo "pensando fora da caixa". Talvez até incluir entre parenteses "direto da caixa (do inglês 'out of the box')". 

```suggestion
Vamos dar uma olhada em algumas das coisas que o Rails fornece "direto da caixa"
```